### PR TITLE
enh(tools/release): release script supports enh prefix

### LINKED ISSUE #2618: Teach release script about "enhancement"
The release script doesn't know how to treat commits prefixed with "enhancement"... they should be treated the same as "feat". The kids are also using "enh()" as a shorthand, I hear. We might want to support that too because "enhancement" eats like half a commit message.

### PR DESCRIPTION
**Pull request by janiceshiu**
Closes #2618

We didn't really agree on what prefix to support, so I'm going with `enh` only for now. Let me know if y'all prefer other options. Thanks!

### CODE REVIEW

File: tools/release.js [hunk 1]
```diff
@@ -191,7 +192,7 @@ const Prompts = {
         if (/^breaking/i.test(line)) {
           return "major";
         }
-        if (/^feat/i.test(line)) {
+        if (/^feat/i.test(line) || /^enh/i.test(line)) {
```
[the remark this answers is not in the document]
Comment (saschanaz) [reply tools/release.js]: As I said in the discussion, I don't think we need enh as every enhancement is either a fix, feat or refactor.

[the remark this answers is not in the document]
Comment (sidvishnoi) [reply tools/release.js]: Oh. Missed that.
So, this PR isn't needed, right?

[the remark this answers is not in the document]
Comment (saschanaz) [reply tools/release.js]: Yup

**Comment by saschanaz** [comment]
Closing as we don't need this 👀

**Pull request closed without merging**